src/components/index.js

```diff
@@ -11,6 +11,10 @@ import MovieDetails from "./movieDetails/MovieDetails";
 import SortControl from "./sortControl/SortControl";
 import Dialog from "./dialog/Dialog";
 import MovieForm from "./movieForm/MovieForm";
+import MovieListPage from "../containers/movieListPage/MovieListPage";
+import Footer from "./footer/Footer";
+import Logo from "./logo/Logo";
+import SearchForm from "./searchForm/SearchForm";

```

Comment:
I'd prefer see here and above something like import { SearchForm } from "./searchForm";

Reply:
I just import them here to later on import all components from one place like ../components